Core/Inc/shared.h

```diff
@@ -26,6 +28,14 @@ typedef struct
 	float maxBoardTemp;
 	float minBoardTemp;
 	float avgBoardTemp;
+
+	float current;
+
+	char* stateMessage;
+
+	char* alertMessage;
+	uint32_t alertIndex;
```

Comment:
Maybe rename this to curAlertIndex? Also add comments describing what each of these variables is